R/conclass.R

```diff
@@ -397,9 +401,34 @@ Conos <- R6::R6Class("Conos", lock_objects=F,
       } else if (is.null(groups) && is.null(colors) && is.null(gene)) {
         groups <- getClusteringGroups(self$clusters, clustering)
       }
+
+      if (!is.null(embedding.type)){
```

Comment:
This `if` shouldn't be here: the logic is valid only when `use.common.embedding=T`, otherwise it extracts embeddings from individual samples. So it should be under `if(use.common.embedding)`.

Reply:
> the logic is valid only when use.common.embedding=T

Thanks, I didn't realize this. I'll remove